addons/binding/org.openhab.binding.systeminfo.test/src/test/groovy/org/openhab/binding/systeminfo/test/SysteminfoOSGiTest.groovy

```diff
@@ -782,7 +1177,15 @@ class SysteminfoOSGiTest extends OSGiTest{
         //The pid of the System idle process in Windows
         int pid = 0
 
+        DecimalType mockProcessMemory = new DecimalType(450)
+        when(mockedSystemInfo.getProcessMemoryUsage(pid)).thenReturn(mockProcessMemory)
+
         initializeThingWithChannelAndPID(channnelID,acceptedItemType,pid)
+
+        ChannelUID processMemoryUsedChannelUID = systemInfoThing.getChannel(SysteminfoBindingConstants.CHANNEL_PROCESS_MEMORY).getUID()
+        SysteminfoHandler thingHandler = getThingHandler(SysteminfoHandler.class)
+
+        assertEquals(mockProcessMemory, thingHandler.getInfoForChannel(processMemoryUsedChannelUID))
         assertItemState(acceptedItemType, DEFAULT_TEST_ITEM_NAME, DEFAULT_CHANNEL_TEST_PRIORITY)
     }
 
```

Comment:
After your changes the `PlatformDependentTestsInterface` will not be needed any more ! :)
This means that you can delete the interface and the `Category` annotations.
Take a look at the pom.xml, the category is used there as well.